org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/AssociateGrammarCodeLensExtensionsTest.java

```diff
@@ -39,6 +39,22 @@ public void noGrammarWithAssociationSupport() throws BadLocationException {
 				cl(r(1, 1, 1, 4), "Bind with xml-model", SELECT_FILE));
 	}
 
+	@Test
+	public void noGrammarWithAssociationSupportInXSD() throws BadLocationException {
+		String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\r\n" + //
+				"<schema />";
+		testCodeLensFor(xml, "test.xsd", //
+				Collections.singletonList(CodeLensKind.Association));
+	}
+
+	@Test
+	public void noGrammarWithAssociationSupportInDTD() throws BadLocationException {

```

Comment:
I don't think this test is working properly, because I still see the CodeLens in the language server trace for a dtd file

Reply:
Ignore, the result was for a references CodeLens